metadata/network.go

```diff
@@ -36,15 +37,17 @@ var TestnetDefinition = NetworkDefinition{
 	"nats://testnet-broker.mysterium.network",
 	"https://ropsten.infura.io",
 	"https://testnet-morqa.mysterium.network/api/v1",
+	"http://metrics.mysterium.network:8091",
```

Comment:
I'm not a fan of sending events directly to ElasticSearch. There are several reasons for that:
1) If we have open ElasticSearch to everyone, it can be easily overloaded, no matter how many nodes we have;
1) If we will need to replace ElasticSearch with some other solution one day, we will need to update every client for that;
1) Any one can fake/compromise these stats. 

I'd do some simple own service that will receive events and send it to the ElasticSearch. In this case we can control access and replace backend db(ElasticSearch) at any moment. 
It could be a part of discovery service or another one.

Reply:
What we could also consider is adding an endpoint to current morqa.

Reply:
We'll agreed on creating this as a separate service. Created separate task for that: https://github.com/mysteriumnetwork/node/issues/701.
We discussed this with @zolia, that it can be done separately, because we're already sending statistics directly to ElasticSearch in all desktop and mobile apps, so we'll need to migrate all those as well - so let's do that separately in other PRs, step by step :)